src/main/scala/Day3.sc

```diff
@@ -0,0 +1,122 @@
+//FUNCTIONAL 1
+val countryCapital = java.util.TimeZone.getAvailableIDs()
+
+countryCapital.map(_.split('/').toList.drop(1).mkString).filter(_.length>1)
```

Comment:
**java.util.TimeZone.getAvailableIDs().map(_.split('/').toList.drop(1).mkString).filter(_.length>1)** no need to have a a variable